pkg/apis/pipeline/v1beta1/task_types.go

```diff
@@ -172,7 +172,7 @@ type Step struct {
 
 // Sidecar has nearly the same data structure as Step, consisting of a Container and an optional Script, but does not have the ability to timeout.
 type Sidecar struct {
-	corev1.Container `json:",inline"`
+	SidecarContainer `json:",inline"`
```

Comment:
The idea was to move all that is to not have this inlined/embedded struct but have the field directly in. So no need for `SidecarContainer`.